atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameUnmapper.java

```diff
@@ -63,6 +66,28 @@ public String getShortTableNameFromMappingTable(
         }
     }
 
+    public Set<String> getLongTableNamesFromMappingTable(
+            ConnectionSupplier connectionSupplier, Set<String> shortTableNames) throws TableMappingNotFoundException {
+        if (shortTableNames.isEmpty()) {
```

Comment:
Oracle fails with an IN () query (interestingly, Sqlite does not!). There are tests for this later in the Oracle integration tests for the namespace cleaner

Reply:
Oracle has a 1000 element IN clause limit, though I believe you can OR multiple IN clauses together